zenith_utils/src/http/endpoint.rs

```diff
@@ -143,6 +154,22 @@ pub fn check_permission(req: &Request<Body>, tenantid: Option<ZTenantId>) -> Res
     }
 }
 
+// Send shutdown signal
+pub fn shutdown() {
+    let f = async {
+        if let Some(tx) = SHUTDOWN_TX.lock().await.take() {
+            let _ = tx.send(());
+        }
+    };
+
+    let runtime = tokio::runtime::Builder::new_current_thread()
```

Comment:
Religious personal opinion: having two separate runtimes to manage the same "entity" as the http server is a bit weird IMO.
I wonder, if some `lazy_static!`-shared runtime would make more sense.